### PR TITLE
Update podspec file

### PR DESCRIPTION
**Pull request by koji-1009**
Update.

### CODE REVIEW

File: StudyplusSDK-V2.podspec [hunk 1]
```diff
@@ -3,13 +3,12 @@ Pod::Spec.new do |s|
   s.version               = "2.0.1"
   s.summary               = "StudyplusSDK-V2 is Studyplus iOS SDK for Swift"
   s.homepage              = "https://info.studyplus.jp"
-  s.license               = { :type => "MIT", :file => "LICENSE" }
+  s.license               = "MIT"
   s.source                = { :git => "https://github.com/studyplus/Studyplus-iOS-SDK-V2.git", :tag => s.version }
   s.source_files          = "StudyplusSDK", "Lib/StudyplusSDK/**/*.{swift}"
-  s.requires_arc          = true
```
Comment (koji-1009) [inline StudyplusSDK-V2.podspec:9]: > The default value of this attribute is true.

https://guides.cocoapods.org/syntax/podspec.html#requires_arc

File: StudyplusSDK-V2.podspec [hunk 2]
```diff
@@ -3,13 +3,12 @@ Pod::Spec.new do |s|
   s.version               = "2.0.1"
   s.summary               = "StudyplusSDK-V2 is Studyplus iOS SDK for Swift"
   s.homepage              = "https://info.studyplus.jp"
-  s.license               = { :type => "MIT", :file => "LICENSE" }
```
Comment (koji-1009) [inline StudyplusSDK-V2.podspec:6]: Simplify.

https://guides.cocoapods.org/syntax/podspec.html#license

File: StudyplusSDK-V2.podspec [hunk 3]
```diff
@@ -3,13 +3,12 @@ Pod::Spec.new do |s|
...
+  s.license               = "MIT"
   s.source                = { :git => "https://github.com/studyplus/Studyplus-iOS-SDK-V2.git", :tag => s.version }
   s.source_files          = "StudyplusSDK", "Lib/StudyplusSDK/**/*.{swift}"
-  s.requires_arc          = true
   s.platform              = :ios, '11.0'
   s.ios.deployment_target = '11.0'
   s.ios.frameworks        = ['UIKit', 'Foundation']
   s.author                = { 'Studyplus inc' => 'developer-all@studyplus.jp' }
-  s.swift_versions        = ['4.2', '5.0']
```
Comment (koji-1009) [inline StudyplusSDK-V2.podspec:14]: Target 5.1 and above.

https://github.com/kean/Nuke/blob/9.3.1/Nuke.podspec#L15

**Review by m-yamada04 — APPROVED** [review]
LGTM:bow:

**Review by nappannda — APPROVED** [review]
LGTM〜〜〜

**Review by koishi — APPROVED** [review]
🙆🏼‍♂️

**Pull request merged**